sys/fido2/ctap/Kconfig

```
@@ -0,0 +1,42 @@
+# Copyright (C) 2021 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+menuconfig MODULE_FIDO2_CTAP
+    bool "FIDO2 CTAP"
+    depends on TEST_KCONFIG
+    help
+        Configure a FIDO2 CTAP authenticator via KConfig.
+
+if MODULE_FIDO2_CTAP
```

To show the options when not using Kconfig dependency resolution, you should instead:
```suggestion
menuconfig KCONFIG_USEMODULE_FIDO2_CTAP
    bool "FIDO2 CTAP"
    depends on USEMODULE_FIDO_CTAP
    help
        Configure a FIDO2 CTAP authenticator via KConfig.

if KCONFIG_USEMODULE_FIDO2_CTAP
```